src/BulletDynamics/Featherstone/btMultiBodyDynamicsWorld.cpp

```diff
@@ -579,6 +579,7 @@ void btMultiBodyDynamicsWorld::integrateMultiBodyTransforms(btScalar timeStep)
 		for (int b = 0; b < m_multiBodies.size(); b++)
 		{
 			btMultiBody* bod = m_multiBodies[b];
+            bod->addSplitV();
```

Comment:
can this be moved after the sleeping check?

Reply:
Done.